cosmwasm/utils.js

```diff
@@ -503,79 +529,43 @@ const makeInstantiateMsg = (contractName, chainName, config) => {
     const { [contractName]: contractConfig } = contracts;
 
     switch (contractName) {
-        case 'Coordinator': {
-            if (chainConfig) {
-                throw new Error('Coordinator does not support chainName option');
-            }
-
+        case CONTRACT_NAME_COORDINATOR: {
             return makeCoordinatorInstantiateMsg(contractConfig, contracts);
```

Comment:
Pass in options to the fn, it can extract the chainConfig internally if it needs to (you can use getChainConfig and make it fail if it's not found), but by default `chainName` is optional